src/app/wallet/wallet.component.ts

```diff
@@ -18,36 +17,39 @@ import {Router} from '@angular/router'
   styleUrls: ['./wallet.component.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class WalletComponent implements OnInit {
+export class WalletComponent {
+  authProviderType = AuthProvider
+  transactionType = TransactionType
 
-  authProvider      = AuthProvider
-  isLoggedIn$       = this.sessionQuery.isLoggedIn$
-  transactionType   = TransactionType
-  authProvider$     = this.sessionQuery.authProvider$
-  copyLabelSub      = new BehaviorSubject<string>("Copy")
-  copyLabel$        = this.copyLabelSub.asObservable()
-  userIdentitySub   = new Subject<String>()
-  userIdentity$     = this.userIdentitySub.asObservable()
```

Comment:
Try to separate different groups of variables with empty lines, like types, subjects with observables, etc. Don't just mix them together with no order.